Gems/AtomLyIntegration/AtomBridge/Code/Source/AtomDebugDisplayViewportInterface.cpp

```diff
@@ -1025,83 +1025,57 @@ namespace AZ::AtomBridge
         if (m_auxGeomPtr &&  radius > FLT_EPSILON &&  axis.GetLengthSq() > FLT_EPSILON)
         {
             AZ::Vector3 axisNormalized = axis.GetNormalizedEstimate();
-            SingleColorStaticSizeLineHelper<(16+1) * 5> lines; // 360/22.5 = 16, 5 possible calls to CreateArbitraryAxisArc
-            AZ::Vector3 radiusV3 = AZ::Vector3(radius);
-            float stepAngle = DegToRad(22.5f);
-            float Deg0 = DegToRad(0.0f);
 
+            const float scale = GetCurrentTransform().RetrieveScale().GetMaxElement();
+            const AZ::Vector3 worldCenter = ToWorldSpacePosition(center);
+            const AZ::Vector3 worldAxis = ToWorldSpaceVector(axis);
 
             // Draw cylinder part (or just a circle around the middle)
             if (heightStraightSection > FLT_EPSILON)
             {
-                DrawWireCylinder(center, axis, radius, heightStraightSection);
-            }
-            else
-            {
-                float Deg360 = DegToRad(360.0f);
-                CreateArbitraryAxisArc(
-                    lines,
-                    stepAngle,
-                    Deg0,
-                    Deg360,
-                    center,
-                    radiusV3,
-                    axisNormalized
-                    );
+                m_auxGeomPtr->DrawCylinderNoEnds(
+                    worldCenter,
+                    worldAxis,
+                    scale * radius,
+                    scale * heightStraightSection,
+                    m_rendState.m_color,
+                    AZ::RPI::AuxGeomDraw::DrawStyle::Line,
+                    m_rendState.m_depthTest,
+                    m_rendState.m_depthWrite,
+                    m_rendState.m_faceCullMode,
+                    m_rendState.m_viewProjOverrideIndex
+                );
```

Comment:
Suggestion: Add a DrawWireCylinderNoEnds to the DebugDisplayRequestBus, Other teams will probably find it useful as well.

Reply:
Will fix in a quick follow up PR.